### DISCUSSION TITLE
The block hash does not match

### DISCUSSION THREAD

**Discussion by amncjf** [filecoin-project/lotus#10904] [General Q&A]
On the filecoin network, use https://api.node.glif.io

I get a partition through 0xd4c16a5518d79986d24292ea12b0eaed31b14ffefd7c0bb84dd37c95142eac3c and a storage partition obtained by taking 2795069. 
The blocks will be published, and the blockHash of their logs is different from the hash of the block.

**Comment by TippyFlitsUK** [comment]
Please use the issue templates provided to submit new tickets @amncjf!
I would also suggest asking this in https://github.com/filecoin-project/ref-fvm
Many thanks 🙏

**Comment by amncjf** [comment]
I use new issue templates to resubmit the issue, also gave the issue to https://github.com/filecoin-project/ref-fvm
Thanks🤝
